apps/src/templates/instructions/topInstructionsDataApi.js

```diff
@@ -37,3 +37,14 @@ export function incrementVisitCount(latestFeedbackId, token) {
     headers: {'X-CSRF-Token': token}
   });
 }
+
+export function getCodeReviewCommentsForProject(channelId, projectVersion) {
+  return $.ajax({
+    url: `/code_review_comments/project_comments`,
```

Comment:
I'd recommend moving this and the other comments ajax call to their own helper library. This would keep all the code for codeReviews in the same place and keep it isolated from other functionality in topInstructions. This makes codeReview a bit more maintainable as changes to its code will have no effect on other portions of topInstructions. It'll also be more versatile as it won't be tied to functionality in topInstructions.